test/new-e2e/tests/process/windows_test.go

```diff
@@ -132,6 +132,9 @@ func (s *windowsTestSuite) TestManualProcessDiscoveryCheck() {
 }
 
 func (s *windowsTestSuite) TestManualProcessCheckWithIO() {
+	s.T().Skip("skipping due to flakiness")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
As a nice to have you can log the ID of the JIRA card that tracks the investigation

Reply:
Done!